package.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Delete package.json and package-lock.json in the root folder. They should only exist in frontend app